### DISCUSSION TITLE
See the current conversation when you reply?

### DISCUSSION THREAD

**Discussion by SebastianStehle** [bkaankose/Wino-Mail#111] [Ideas]
Hi,

is there a setting to see the current conversation when you reply to a message? Or is this not implemented yet?

**Comment by bkaankose** [comment]
Conversation threads are automatically enabled. Each reply must be visible under the message they've been sent to by default.

**Reply by SebastianStehle** [reply]
When I reply to a message I do not see the original message.

**Comment by bkaankose** [comment]
This is fixed for sometime. Replies already included.
